docs/proposals/004-endpoint-picker-protocol/README.md

```diff
@@ -57,6 +63,25 @@ Constraints:
 - The EPP MUST not set two different values in the header and the inner response metadata value.
 - Setting different value leads to unpredictable behavior because proxies aren't guaranteed to support both paths, and so this protocol does not define what takes precedence.
 
+## Destination Endpoint Status
+
+[RESPONSE: Data Plane -> EPP]
+
+For each HTTP response, the data plane MUST communicate to the EPP endpoint status as follows:
+
+In the ext_proc [ProcessingRequest.metadata_context](https://github.com/envoyproxy/envoy/blob/v1.35.0/api/envoy/service/ext_proc/v3/external_processor.proto#L130) field, a new metadata field "x-gateway-destination-endpoint-status" is added as an unstructured entry with an outer key (which represents the metadata namespace) with a default value of `envoy.lb`:
+
+```go
+filterMetadata: {
+  "envoy.lb": {
+    "x-gateway-destination-endpoint-status": endpoint=<ip:port>,
+}
+```
+
+The value associated with this metadata field is a comma delimited string. The following status values are supported:
+
+- endpoint=ip:port: the endpoint which received the request; depending on endpoint health and connectivity status, the data plane may have had to attempt multiple endpoints (see [Destination Endpoint](#destination-endpoint)).
+
```

Comment:
I think we would like to know what is the final endpoint that served the request successfully. 
we don't care about the ones that had an attempt and failed due to possible reasons.
so this field should be in the form of `ip:port` (only one). 
also, to be consistent with other points in the protocol, I think we don't need the `endpoint=`.
so it's just:
```go
filterMetadata: {
  "envoy.lb": {
    "x-gateway-destination-endpoint-status": ip:port
  }
}
```